ote/src/cljs/ote/views/main.cljs

```diff
@@ -66,20 +69,23 @@
                 :style style-base/flash-message
                 :auto-hide-duration 5000}])
 
-(def header-links
-  [{:page :front-page
-    :label [:common-texts :navigation-front-page]
-    :url "/"}
+(defn header-links [app]
+  (filter some?
+          (conj
```

Comment:
You don't need the `conj`. You can just `(filter some? [first-item second-item (when condition? third-item)])`

Reply:
True. Forgot to remove it after adding filter.